.github/workflows/test.yml

```diff
@@ -5,11 +5,12 @@ on:
 
 jobs:
   test:
-    name: Test on Python ${{ matrix.python-version }}
+    name: ${{ matrix.test-type }} tests on Python ${{ matrix.python-version }}
     runs-on: ubuntu-latest
     strategy:
       matrix:
         python-version: [ '3.11' ]
+        test-type: ["Unit", "Migration"]
```

Comment:
Is this required? What if we skipped `test-type` matrix entry and just had two sequential steps, one `Unit tests` and the other `Migration tests`?

Reply:
It's not required, but the migration tests take some time to run (~3 minutes for just the ones that were added in this PR). Separating them into the parallel job keeps the whole suite a bit shorter. But I wouldn't mind reverting this for now and re-assessing later if it becomes a problem.